test/unit/data/events/scheme/group-events-by-scheme.test.js

```diff
@@ -0,0 +1,34 @@
+const { BPS, CS, SFI } = require('../../../../../app/constants/schemes')
+const submitted = require('../../../../mocks/events/submitted')
```

Comment:
I'd put the mock at the top of the file with a space between to separate it from the constant